Snippets/Core/Core_6/Msmq/DLQOptInForTTBR.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Core6.Msmq
+{
+    using NServiceBus;
+
+    class DLQOptInForTTBR
+    {
+        DLQOptInForTTBR(EndpointConfiguration endpointConfiguration)
+        {
+            #region msmq-dlq-for-ttbr-optin
+
+            endpointConfiguration.UseTransport<MsmqTransport>()
+                .UseDeadLetterQueueForMessagesWithTimeToReachQueue();
```

Comment:
@andreasohlund - thanks. 
According to this article, there are 2 ways that messages can reach DLQ. When the TTBR or TTRQ timers elapse. This API talks about TTRQ. But the heartbeat etc, don't we set the TTBR?
https://blogs.msdn.microsoft.com/johnbreakwell/2010/01/22/where-have-my-msmq-messages-gone/